### PR TITLE
hyundai fw sort

### PR DESCRIPTION
**Pull request by crwusiz**
interface fix

fw sort
fwdRadar, fwdCamera, eps, esp, engine, transmission

### CODE REVIEW

**Comment by pd0wm** [comment]
I understand you want to clean things up, but the diff to `selfdrive/car/hyundai/values.py` is too large to be worth it. Those FW versions will be moved out of values.py at some point anyway.

I'd merge the cleanup to `interface.py` though!

File: selfdrive/car/hyundai/interface.py [hunk 1]
```diff
@@ -191,7 +182,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.steerRatio = 13.75
       tire_stiffness_factor = 0.5
       ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0.], [0.]]
-      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
+      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
```
Comment (pd0wm) [inline selfdrive/car/hyundai/interface.py:185]:
```suggestion
      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
```

**Comment by pd0wm** [comment]
Can you revert the changes to values.py?

**Comment by crwusiz** [comment]
> Can you revert the changes to values.py? Looks like I don't have push access to your fork.

value change revert...now

**Pull request merged**